$NetBSD: patch-bj,v 1.1 2006/10/04 21:07:52 rillig Exp $

Without this patch, some Python scripts had their interpreter changed to
python2.1None instead of python2.1.

--- Lib/distutils/command/build_scripts.py.orig	2001-07-30 11:56:31.000000000 +0200
+++ Lib/distutils/command/build_scripts.py	2006-10-04 22:59:10.000000000 +0200
@@ -83,6 +83,8 @@ class build_scripts (Command):
                 if match:
                     adjust = 1
                     post_interp = match.group(1)
+                    if post_interp == None:
+                        post_interp = ""
 
             if adjust:
                 self.announce("copying and adjusting %s -> %s" %
